src/patternfly/components/Card/examples/card-no-footer-example.hbs

```diff
@@ -1,8 +1,8 @@
-{{#> card pf-c-card-modifier="" }}
-  {{#> card-header pf-c-card__header-modifier="" }}
+{{#> card}}
+  {{#> card-header }}
```

Comment:
remove space after header :)

Reply:
fixed

Reply:
To see if you have any more of these, you can search the codebase for ` }}` and `{{ `

Reply:
Oops I don't think the code block above worked. I mean search for `[space]}}` and `{{[space]`